contact_search_form/__manifest__.py

```diff
@@ -0,0 +1,35 @@
+#####################################################################
+#
+#    IT IS AG, software solutions: http://www.itis.de
+#    License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+#
+#####################################################################
+{
+    'name': "Contact Search Form",
+
+    'summary': """
```

Comment:
Please put a shorter summary (this is equal to the description), because this is shown on the module kanban view and then it's cut without seeing full text. You should also avoid triple quote, as it keeps the line feeds and spaces, and makes this text even longer.